src/main/java/jenkins/plugins/office365connector/model/Macro.java

```diff
@@ -40,8 +40,8 @@
 
     @DataBoundConstructor
     public Macro(String template, String value) {
-        this.template = Util.fixEmptyAndTrim(template);
-        this.value = Util.fixEmptyAndTrim(value);
+        this.template = Util.fixNull(template);
```

Comment:
Empty spaces might be valid sometimes and apply ``trim()`` might not be good option. For sure we don't want o accept null values